modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js

```diff
@@ -114,7 +114,7 @@ const CreationMenu = ({
 				symbolLeft={item.icon}
 				{...getDataAttributes(item.data)}
 			>
-				{item.label}
+				{Liferay.Util.unescapeHTML(item.label)}
```

Comment:
I am was hesitant about passing an unescaped value to be rendered, thinking of something like `<b>hello<script>alert('hi')</script></b>`.

However, I think because this is being passed into JSX, we should be okay because JSX automatically protects against xss.

Reply:
Hi @bryceosterhaus Thank you for spending time on this. I tested your case and the solution works well. I don't see anything out of the ordinary. Thanks!